pkg/scene/scan.go

```diff
@@ -333,3 +357,48 @@ func getInteractive(path string) bool {
 	_, err := os.Stat(GetFunscriptPath(path))
 	return err == nil
 }
+
+func getCaption(path string) string {
+	captions := ""
+	parent := filepath.Dir(path)
+	baseName := filepath.Base(path)
+
+	// only proceed if we haven't already looked at this directory
+	if previousDir != parent {
+		files, err := ioutil.ReadDir(parent)
+		if err != nil {
+			return captions
+		}
+		// if directory contains more file than specified value assume no organization
```

Comment:
While this is good for optimization it requires a certain folder structure from the user. I will have a look if we can move the scan code outsite the scene so we can detect all subtitles more efficiently